apps/src/code-studio/components/Libraries/LibraryCreationDialog.jsx

```diff
@@ -67,37 +103,97 @@ class LibraryCreationDialog extends React.Component {
     );
   };
 
+  validateInput = () => {
+    // Check if any of the checkboxes are checked
+    // If this changes the publishable state, update
+    let formElements = document.getElementById('selectFunction').elements;
+    let isChecked = false;
+    [...formElements].forEach(element => {
+      if (element.type === 'checkbox' && element.checked) {
+        isChecked = true;
+      }
+    });
+    if (isChecked !== this.state.canPublish) {
+      this.setState({canPublish: isChecked});
+    }
+  };
+
   displayFunctions = () => {
     if (!this.state.loadingFinished) {
-      return <div>Loading...</div>;
+      return <div id="loading">Loading...</div>;
```

Comment:
If you get the chance, it would be nice to make this a spinner instead of `Loading...`

Reply:
https://codedotorg.atlassian.net/browse/STAR-775